<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfAirCombatPrefWidget</class>
 <widget class="QWidget" name="WkfAirCombatPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Air Combat Visualization</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QComboBox" name="mSecondMeterBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mThirdMeterLabel">
     <property name="text">
      <string>Third Meter</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="mShowStateData">
     <property name="text">
      <string>Show State Data</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="mShowWeaponsAccent">
     <property name="text">
      <string>Show Weapons Accent</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="mShowTransmissionAccent">
     <property name="text">
      <string>Show Transmission Accent</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="mShowSignatureAccent">
     <property name="text">
      <string>Show Signature Accent</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>254</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mSecondMeterLabel">
     <property name="text">
      <string>Second Meter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mFirstMeterLabel">
     <property name="text">
      <string>First Meter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="mFirstMeterBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="mShowFuelAccent">
     <property name="text">
      <string>Show Fuel Accent</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="mThirdMeterBox"/>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="mShowVisualization">
     <property name="text">
      <string>Show Air Combat Visualization</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
